sycl/plugins/esimd_cpu/pi_esimd_cpu.cpp

```diff
@@ -1048,18 +1647,25 @@ pi_result piEnqueueMemImageRead(pi_queue command_queue, pi_mem image,
                                 pi_uint32 num_events_in_wait_list,
                                 const pi_event *event_wait_list,
                                 pi_event *event) {
-  (void)command_queue;
-  (void)image;
-  (void)blocking_read;
-  (void)origin;
-  (void)region;
-  (void)row_pitch;
-  (void)slice_pitch;
-  (void)ptr;
-  (void)num_events_in_wait_list;
-  (void)event_wait_list;
-  (void)event;
-  DIE_NO_IMPLEMENTATION;
+  _pi_image *img = static_cast<_pi_image *>(image);
```

Comment:
It seems check for `blocking_read` is needed here as well.

Reply:
Will apply.

Reply:
Done.